README.md

```diff
@@ -1,21 +1,38 @@
 # ai-fashion-mirror
 
 ## Getting started
+
 This project uses the node v20.16.0 (LTS) and npm 10.8.1.
 
-First, we'll need to install frontend and backend dependencies.
+First, we'll need to install frontend, backend dependencies and root dependencies.
 
 ```shell
+npm i
 cd client
 npm i
 cd ../server
 npm i
 ```
 
+This project uses [Prettier](https://prettier.io/) and [ESLint](eslint.org) for automatic formatting and fixing.
+
+Make sure to configure your [editor](https://prettier.io/docs/en/editors) to allow Prettier/ESLint on save.
+
+1. On VSCode install the Prettier and ESLint extention. Once installed and enabled, you may need to restart VSCode.
```

Comment:
there's a prettier and a prettier eslint, i want to make sure that we're supposed to get prettier and eslint seperately?
<img width="489" alt="Screen Shot 2024-08-15 at 7 46 29 PM" src="https://github.com/user-attachments/assets/c5994573-2ed1-4788-aef0-624cba7faae5">

Reply:
yes, two separate extensions, sorry for the confusion!
**Prettier - Code formatter** and **ESLint**

Reply:
updated the readme to make it more clear!